src/lib/requestLogger.js

```diff
@@ -27,15 +27,15 @@
 const Logger = require('@mojaloop/central-services-shared').Logger
 const Util = require('util')
 
-const logRequest = function (request) {
-  Logger.debug(`ALS-Trace - Method: ${request.method} Path: ${request.url.path} Query: ${JSON.stringify(request.query)}`)
-  Logger.debug(`ALS-Trace - Headers: ${JSON.stringify(request.headers)}`)
+const logRequest = function (request, logger = Logger) {
```

Comment:
What is the purpose of passing in the logger as a parameter over having it as an import?

Reply:
Allows one to silence logging in the tests, or any other time, without relying on global state.